packages/components/src/components/InfoItem/InfoItem.tsx

```diff
@@ -50,18 +42,18 @@ export type InfoItemProps = AllowedFrameProps &
         direction?: FlexDirection;
         iconName?: IconName;
         label: ReactNode;
-        labelTypographyStyle?: TypographyStyle;
+        variant?: IconVariant;
```

Comment:
I'm not fan of reusing variants for other components - it can be a bit blackbox when we for example add new variant for icon, but we don't want interfere with other components. 
Why not to create `InfoItemVariant` here?